Source/Programs/wavm.cpp

```diff
@@ -0,0 +1,114 @@
+#include "Core/Core.h"
+#include "Core/MemoryArena.h"
+#include "Core/Platform.h"
+#include "AST/AST.h"
+#include "Runtime/Runtime.h"
+#include "Runtime/RuntimePrivate.h"
+
+#include "CLI.h"
+
+bool endsWith(const char *str, const char *suffix)
+{
+	if (!str || !suffix)
+		return false;
+	size_t lenstr = strlen(str);
+	size_t lensuffix = strlen(suffix);
+	if (lenstr < lensuffix)
+		return false;
```

Comment:
Can you put parentheses around these if-bodies to be consistent with the rest of the code?